.github/infra/triggers/test-webui-master.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: test-webui-master
+description: Test the UI components of the reference app on push to master
+filename: webui/cloudbuild-test.yaml
+includedFiles:
+- webui/**
+- .github/infra/**
+github:
```

Comment:
I wonder if we should start linking to the reference for our yaml files. It would be great if I could click on a link here and read the docs for this.

Reply:
Added a link to each of the YAML files, PTAL

Reply:
Neat - that helps a lot. Thanks!